onnxruntime/core/providers/mkldnn/subgraph/subgraph.h

```diff
@@ -0,0 +1,68 @@
+// Copyright(C) 2019 Intel Corporation
+// Licensed under the MIT License
+
+#pragma once
+
+#include <vector>
+#include <string>
+#include <map>
+
+namespace onnxruntime {
+
+struct MklNode {
```

Comment:
there's already onnxruntime representations of Node, subgraphs etc.
Were those too heavy to use? Would have been nice to leverage more of onnxruntime classes where possible,
rather than adding new ones to bridge onnxruntime and mkldnn library.

Reply:
MklDnnNode is totally different and I am using it for MklDnn IR of a subgraph.